src/test/ui/fn_once-moved.stderr

```diff
@@ -1,12 +1,15 @@
 error[E0382]: use of moved value: `debug_dump_dict`
   --> $DIR/fn_once-moved.rs:21:5
    |
+16 |         for (key, value) in dict {
+   |                             ---- dict moved here
```

Comment:
I don't know =) I was imagining output like this. We do still support this style, though we've been moving away from it in many cases. This may be a case where it's worthwhile:

```
error[E0382]: use of moved value: `debug_dump_dict`
  --> $DIR/fn_once-moved.rs:21:5
   |
20 |     debug_dump_dict();
   |     --------------- value moved here
21 |     debug_dump_dict();
   |     ^^^^^^^^^^^^^^^ value used here after move
   |
note: closure cannot be invoked more than once because it moves the variable `dict` out of its environment
    |
16 |         for (key, value) in dict {
   |                             ---- dict moved here
```

I think that if you use `span_note` it will produce something sort of like this? You may have to experiment a bit. (I'm also unclear on whether "help" or "note" is more appropriate here.)